lib/pages/chat/events/message.dart

```diff
@@ -515,6 +511,23 @@ class Message extends StatelessWidget {
       ),
     );
   }
+
+  bool isSameSender(Event? previousEvent) {
+    if (previousEvent == null) {
+      return true;
+    }
+
+    final isPreviousEventMessage = {
+      EventTypes.Message,
+      EventTypes.Sticker,
+      EventTypes.Encrypted,
+      EventTypes.Redaction,
+    }.contains(previousEvent.type);
+
+    return isPreviousEventMessage
+      ? previousEvent.senderId != event.senderId
+      : !isPreviousEventMessage;
```

Comment:
hard to understand @@

Reply:
I broke down the ternary and added comments